build-tests/set-webpack-public-path-plugin-webpack4-test/config/heft.json

```diff
@@ -4,48 +4,28 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/heft/heft.schema.json",
 
-  "eventActions": [
-    {
-      /**
-       * The kind of built-in operation that should be performed.
-       * The "deleteGlobs" action deletes files or folders that match the
-       * specified glob patterns.
-       */
-      "actionKind": "deleteGlobs",
-
-      /**
-       * The stage of the Heft run during which this action should occur. Note that actions specified in heft.json
-       * occur at the end of the stage of the Heft run.
-       */
-      "heftEvent": "clean",
-
-      /**
-       * A user-defined tag whose purpose is to allow configs to replace/delete handlers that were added by other
-       * configs.
-       */
-      "actionId": "defaultClean",
-
-      /**
-       * Glob patterns to be deleted. The paths are resolved relative to the project folder.
-       */
-      "globsToDelete": ["dist-dev", "dist-prod", "lib", "temp"]
-    }
-  ],
-
-  /**
-   * The list of Heft plugins to be loaded.
-   */
-  "heftPlugins": [
-    {
-      /**
-       * The path to the plugin package.
-       */
-      "plugin": "@rushstack/heft-webpack4-plugin"
-
-      /**
-       * An optional object that provides additional settings that may be defined by the plugin.
-       */
-      // "options": { }
+  // TODO: Add comments
+  "phasesByName": {
```

Comment:
I've been thinking for a while that we should just use the rigs for these test projects that aren't specifically exercising Heft.

Reply:
I've updated a couple to use the rig instead of specially defining a Heft config, but the one you are pointing to here is a webpack4 test project, whereas the web rig currently has webpack5. We can of course override that plugin to use webpack4, but not sure that gets us much there. Thoughts?

Reply:
Naw, I agree on keeping the webpack4 one as a standalone.